labs/middleware.py

```diff
@@ -67,32 +49,85 @@ def call_llm_with_context(github: GithubModel, issue_summary, litellm_api_key):
             "content": prompt,
         }
     )
-    try:
-        llm_response = litellm_requests.completion_without_proxy(
-            prepared_context,
-            model=LLM_MODEL_NAME,
+    return prepared_context
+
+
+def validate_llm_response(llm_response):
```

Comment:
lets improve this function, perhaps use a dict with all the possible cases mapped out.

Reply:
Changed it. Check if it's good now.